pkg/cosign/manager.go

```diff
@@ -0,0 +1,77 @@
+package cosign
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+type AcornPublicKey struct {
+	ID          string `json:"id,omitempty"`
+	Key         string `json:"key,omitempty"`
+	Fingerprint string `json:"fingerprint,omitempty"`
+}
+
+func getAcornPublicKeys(username string) ([]AcornPublicKey, error) {
+	logrus.Debugf("Getting public keys for Acorn user %s", username)
+
+	parts := strings.Split(username, "@")
+	if len(parts) > 2 {
+		return nil, fmt.Errorf("invalid username: %s", username)
+	}
+
+	host := "https://hub.acornlabs.com"
```

Comment:
nit: I think this should be

```suggestion
	host := "https://acorn.io"
```

Reply:
Absolutely!